overlord/ifacestate/handlers.go

```diff
@@ -458,13 +444,9 @@ func (m *InterfaceManager) doDisconnect(task *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	affectedConns, err := m.repo.ResolveDisconnect(plugRef.Snap, plugRef.Name, slotRef.Snap, slotRef.Name)
```

Comment:
Ok, fixed to error out with such message on Disconnect call. I also realized it's better to call repo.Disconnect() than repo.DisconnectAll(), fixed that as well.